client/src/shared/EditCredentialsInputHandler.tsx

```diff
@@ -0,0 +1,37 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { passwordRegExp } from "./LoginInputHandler";
+
+const editCredentialsSchema = Yup.object({
+  currentPassword: Yup.string()
+    .required("Password is required")
+    .min(4, "Password is too short - should be 4 chars minimum")
+    .matches(
+      passwordRegExp,
+      "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character"
+    ),
+  newPassword: Yup.string()
+    .min(4, "Password is too short - should be 4 chars minimum")
```

Comment:
I think something here is wrong, firstly password must be minimum 4 characters and then 8 characters? :D